public/pages/CreateMonitor/components/LocalUriInput/LocalUriInput.js

```diff
@@ -0,0 +1,60 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React, { Fragment } from 'react';
+import { EuiSpacer, EuiFlexItem, EuiFlexGroup, EuiCodeEditor, EuiFormRow } from '@elastic/eui';
+import { hasError, isInvalid } from '../../../../utils/validate';
+import { FormikFieldText } from '../../../../components/FormControls';
+
+const LocalUriInput = ({ isDarkMode, response, values }) => (
+  <Fragment>
+    <EuiFlexGroup alignItems="flexStart">
+      <EuiFlexItem>
+        <EuiSpacer size="m" />
+        <FormikFieldText
+          name={`${values.searchType}.path`}
+          formRow
+          rowProps={{
+            label: 'Path',
+            helpText: 'Example path: "/_cluster/health/"',
```

Comment:
Minor: Seems the help text is too simple to explain what can be filled in the field. Such as telling the user only Elasticsearch REST APIs can be the input. Well, opinions about the help text is better to be told by a tech writer, so it's up to you. 😂

Reply:
I think this is a good one, it sound more professional. 😄